docs/_includes/markup/tooltips.njk

```diff
@@ -1,12 +1,12 @@
-<button type="button" class="btn btn-ui btn-sm" data-toggle="tooltip" data-placement="top" title="Tooltip on top">
+<button type="button" class="btn btn-ui btn-sm" data-toggle="tooltip" data-trigger="click" data-placement="top" title="Tooltip on top">
```

Comment:
I don't know that these are necessarily making these any more accessible. Bootstrap, by default, will make a tooltip show up on click when on a device that doesn't support hover. All this has done is make it not hover for those who can do so.

Reply:
this is addressed later in the conversation @james-alt